index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+    <link rel="stylesheet" href="index.css">
+    <title>Portifolio page revire</title>
+</head>
+
+<body>
+    <nav id="navbar">
+        <img id="logo" class="dagi" width='150'
+            src="https://avatars.githubusercontent.com/u/68595873?s=400&u=80bbdfc291f01cffaae57bee393b989ea3b24bda&v=4"
+            alt="">
```

Comment:
Kindly, it is better to add a link into an image embed a <a> tag with its correspondent href property. Most web pages use their own logo to go back to the main page, so it would be nice to start practicing doing it.